utils/xmtpRN/sync.ts

```diff
@@ -88,8 +86,6 @@ export const onSyncLost = async (account: string, error: any) => {
   }
 };
 
-const streamingAccounts: { [account: string]: boolean } = {};
```

Comment:
Curious, if there was anywhere else, this was used other than where you deleted it below